docs/deprecated_wiki/Installing-Loris.md

```diff
@@ -1,35 +0,0 @@
-This page provides OS-specific links for installation of the basic LORIS codebase. 
```

Comment:
This has been superseded by various OS install guides https://github.com/aces/Loris/tree/master/docs/wiki/00%20-%20SERVER%20INSTALL%20AND%20CONFIGURATION/01%20-%20LORIS%20Install